doc/Pacemaker_Administration/en-US/Ch-Tools.txt

```diff
@@ -16,19 +16,21 @@ command line options for changing the style of the output.  For instance:
 
 ======
 -------
-# crm_mon --help-output-html
+# crm_mon --help-output
 Usage:
   crm_mon [OPTION?]
 
 Provides a summary of cluster's current state.
 
 Outputs varying levels of detail in a number of different formats.
 
-Output Options (html):
-  --output-cgi                      Add text needed to use output in a CGI program
-  --output-meta-refresh=SECONDS     How often to refresh
-  --output-stylesheet-link=URI      Link to an external CSS stylesheet
-  --output-title=TITLE              Page title
+Output Options:
+  --output-as=FORMAT                Specify output format as one of: console (default), html, text, xml
+  --output-to=DEST                  Specify file name for output (or "-" for stdout)
+  --html-cgi                        Add text needed to use output in a CGI program
+  --html-stylesheet-link=URI        Link to an external CSS stylesheet
+  --html-title=TITLE                Page title
+  --text-fancy                      Use more highly formatted output
```

Comment:
I do think it looks great like this. A tiny tweak, let's drop the "-link" for brevity